vendor/github.com/go-openapi/spec/.golangci.yml

```diff
@@ -21,3 +21,8 @@ linters:
     - lll
```

Comment:
why has openapi been pulled into vendor?

Reply:
haven't pulled it Gopkg.toml, might be some external dependency of chaos--operator api's